examples/reference/streams/bokeh/Tap.ipynb

```diff
@@ -39,23 +112,32 @@
     "\n",
     "# Declare HeatMap\n",
     "heatmap = hv.HeatMap(dataset.aggregate(['Year', 'State'], np.mean),\n",
-    "                     label='Measles Incidence').select(Year=(1928, 2002))\n",
+    "                     label='Average Weekly Measles Incidence').select(Year=(1928, 2002))\n",
     "\n",
     "# Declare Tap stream with heatmap as source and initial values\n",
     "posxy = hv.streams.Tap(source=heatmap, x=1951, y='New York')\n",
     "\n",
     "# Define function to compute histogram based on tap location\n",
     "def tap_histogram(x, y):\n",
     "    return hv.Curve(dataset.select(State=y, Year=int(x)), kdims='Week',\n",
-    "                   label='Year: %s, State: %s' % (x, y))\n",
+    "                   label=f'Year: {x}, State: {y}')\n",
```

Comment:
I forget; is HoloViews py2 only now? If not this change should presumably be reverted.

Reply:
ah. I just couldn't help myself... I'll revert.

Reply:
this has been reverted

Reply:
It doesn't _look_ reverted; was that in a separate PR?